File: Tugas coding modul 2/coding py/PRAK202-2310817120005-Noviana_Nur_Aisyah.py
# TEST CASE KE-1
# Nilai pertama 14
# Nilai kedua 20.5

# TEST CASE KE-2
# Nilai pertama 0.45
# Nilai kedua 99.5

print('Input')
satu = input('Masukkan Nilai Pertama : ')
dua = input('Masukkan Nilai Kedua : ')
tiga = float(satu) + float(dua)

print('\nOutput')
print(f'Hasil dari penjumlahan nilai pertama "{satu}" dan nilai kedua "{dua}" adalah “{tiga:.2f}”')

File: Tugas coding modul 2/coding c/PRAK202-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void main(){

    printf("Input\n");

    float first, second, sum;

    // 14, 20.5 | 0.45, 99.5

    printf("Masukkan Nilai Pertama : ");
    scanf("%f", &first);

    printf("Masukkan Nilai Kedua : ");
    scanf("%f", &second);

    sum = first + second;
    
    printf("\nOutput\n");
    printf("Hasil dari penjumlahan nilai pertama \"%g\" dan nilai kedua \"%g\" adalah \"%.2f\"", first, second, sum);
}